pkg/api/docs/merge/merge_specs.go

```diff
@@ -98,6 +135,61 @@ func mergeSpecs(output string, sources ...string) error {
 	return nil
 }
 
+// getDashboardDefinitions converts the dashboard OpenAPI v3 to OpenAPI v2
+// and it returns it as spec.Schema
+func getDashboardDefinitions() (spec.Definitions, error) {
+	lib := thema.NewLibrary(cuecontext.New())
```

Comment:
this is totally fine for a one-off command like this one. But FYI, `grafana/pkg/cuectx.ProvideThemaLibrary()` is the centrally-defined shared `thema.Library` that, within the Grafana codebase, is usually the one you should opt to use.

The concern here is that certain methods on `cue.Value` will panic if called with a `cue.Value` that's from a different CUE runtime/context. With the way we're using it here, though, that's really just not possible.